smart-contracts/contracts/UnlockDiscountToken.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity 0.5.9;
+
+import 'openzeppelin-eth/contracts/token/ERC20/ERC20Mintable.sol';
+import 'openzeppelin-eth/contracts/token/ERC20/ERC20Detailed.sol';
+
+
+/**
+* @title The Unlock Discount Token
+* This smart contract implements the Unlock Discount Token
+*/
+contract UnlockDiscountToken is
+ERC20Mintable,
+ERC20Detailed
+{
+    function initialize() public initializer {
```

Comment:
Lint error on this.  `solium` is configured for 2 spaces.  Could you fix?

From CI:
```
contracts/UnlockDiscountToken.sol
  15:4    error    Only use indent of 2 spaces.    indentation
  16:8    error    Only use indent of 6 spaces.    indentation
  17:8    error    Only use indent of 6 spaces.    indentation
  18:0    error    Only use indent of 2 spaces.    indentation
```